instance/models/utils.py

```diff
@@ -482,6 +484,49 @@ def purge(self):
         """
         return self._client.kv.delete(self.prefix, recurse=True)
 
+    def leave(self, force=True):
```

Comment:
This is dangerous to be `True` by default. If we somehow managed to run this on the Consul server agents, we could have a problem. See 2nd paragraph in [the docs](https://www.consul.io/api/agent.html#graceful-leave-and-shutdown).